api/lib/block-listeners/polkadot-node-subscription.js

```diff
@@ -223,6 +236,13 @@ class PolkadotNodeSubscription {
       Sentry.captureException(error)
     }
   }
+
+  async storeNetworkInDB() {
+    // only run at startup
+    await this.store.storeNetwork(this.store.network)
```

Comment:
why are we storing the network in the db? doesn't it need to be in there already at startup?

Reply:
It is a good idea. Imagine developing and adding a new network. The thing is that this part is automatic, storing the network in the DB. You only need to fill the network specs in data, and the rest will be automatically handled. And then, if we need to change something on the go, we do it simply using the DB

Reply:
that doesn't seem right. we would overwrite the persisted data. let's not manage the data twice. for development you could have a local networks.js file which adds development networks.

Reply:
OK, I see what you mean